src/Notifications/notificationSections.test.jsx

```diff
@@ -83,16 +84,23 @@ describe('Notification sections test cases.', () => {
   });
 
   it('Successfully load more notifications by clicking on load more notification button.', async () => {
+    axiosMock.onPut(markNotificationsSeenApiUrl('discussion')).reply(200);
+    await executeThunk(markNotificationsAsSeen('discussions'), store.dispatch, store.getState);
     renderComponent();
 
     const bellIcon = screen.queryByTestId('notification-bell-icon');
     await act(async () => { fireEvent.click(bellIcon); });
 
+    expect(screen.queryAllByTestId('notification-contents')).toHaveLength(10);
     const loadMoreButton = screen.queryByTestId('load-more-notifications');
 
-    expect(screen.queryAllByTestId('notification-contents')).toHaveLength(10);
-    await act(async () => { fireEvent.click(loadMoreButton); });
+    axiosMock.onGet(getNotificationsListApiUrl()).reply(
+      200,
+      (Factory.build('notificationsList', { num_pages: 2, current_page: 2 })),
+    );
+    await executeThunk(fetchNotificationList({ appName: 'discussion', page: 2 }), store.dispatch, store.getState);
```

Comment:
can it mock after the component render?

Reply:
after component rendering, load more will hit API again to fetch data, which will bring response of the most recent mock